VoodooI2C/VoodooI2C/VoodooI2CDevice/VoodooI2CDeviceNub.cpp

```diff
@@ -148,16 +148,31 @@ IOReturn VoodooI2CDeviceNub::getAlternativeInterrupt(VoodooI2CACPICRSParser* crs
 
     data->release();
 
-    if (!crs_parser->found_gpio_interrupts) {
-        IOLog("%s::%s Failed to find valid interrupts from _DSM or XDSM method\n", getName(), acpi_device->getName());
+    IOLog("%s::%s Found valid resources from _CRS method\n", getName(), acpi_device->getName());
+
+    return kIOReturnSuccess;
+}
+
+IOReturn VoodooI2CDeviceNub::parseResourcesDSM(VoodooI2CACPICRSParser* crs_parser) {
```

Comment:
This can be set up as pass by reference than a pointer. It would ensure that `crs_parser` will always be present and not NULL.

Reply:
Done